sdk/highlight-dotnet4/OpenTelemetry.cs

```diff
@@ -107,7 +107,7 @@ public static Dictionary<string, object> GetResourceAttributes()
                 ["highlight.project_id"] = Cfg.ProjectId,
                 ["service.name"] = Cfg.ServiceName,
                 ["telemetry.distro.name"] = "Highlight.ASP4",
-                ["telemetry.distro.version"] = "0.1.7",
+                ["telemetry.distro.version"] = "0.2.0-beta.7",
```

Comment:
Think the warnings about `Serilog` are relevant?

Reply:
which ones?